samples/precache/catsrc-child-policy.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: policy.open-cluster-management.io/v1
```

Comment:
For samples, would it be worth including the condensed subscription policy? I've also included it in one of the KUTTL testcases: https://github.com/openshift-kni/cluster-group-upgrades-operator/pull/87/files#diff-413fcbbd4332dc448f6267751c3c9ebd0dd4214da59a4069b68f49eab514df8f.

Maybe the group-du-sno-validator-du-validator-policy also? Just to make sure we're covering a bunch of different policy formats.

Reply:
There is a subscription policy (Line 40):
https://github.com/openshift-kni/cluster-group-upgrades-operator/pull/85/files#diff-35155d9baeb86ab3c1c93fd8a3f226eae04c83605802a4673ae3566c944ea273
I added only policies relevant for pre-caching, since these are pre-caching specific tests. Maybe we should think about adding e2e tests that will test pre-caching and upgrade together? If / when we do, I agree it would make much more sense adding all the policy kinds you've mentioned.